src/scripts.js

```diff
@@ -20,6 +20,7 @@ import './images/flame.png'
 import userData from './data/users';
 import UserRepository from './UserRepository';
 import User from './User';
+import {getUserData, getSleepData, getActivityData, getHydrationData} from './apiCalls';
```

Comment:
Note to see if we can play around with the one object import here if we have time